python/setup.py

```diff
@@ -560,9 +560,8 @@ def build(build_python, build_java, build_cpp):
         )
 
     bazel_flags = ["--verbose_failures"]
-    if BAZEL_LIMIT_CPUS:
-        n = int(BAZEL_LIMIT_CPUS)  # the value must be an int
-        bazel_flags.append(f"--local_cpu_resources={n}")
+    if BAZEL_ARGS:
+        bazel_flags.extend(BAZEL_ARGS.split(" "))
```

Comment:
could you use `shlex.split(BAZEL_ARGS)` to parse here?

Reply:
Oh, nifty - thanks for this!